frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -1,9 +1,42 @@
+import { modal } from '../../../integration-tests-cypress/views/modal';
 import { checkErrors } from '../../../integration-tests-cypress/support';
-import { storagePool, PoolState } from '../views/multiple-pool';
+import { POOL_PROGRESS } from '../../src/constants/storage-pool-const';
+import {
+  poolName,
+  populateBlockPoolform,
+  blockPoolFooter,
+  deleteBlockPoolFromCli,
+  verifyBlockPool,
+} from '../views/block-pool';
 
 // Pool var
-const poolName: string = 'example.pool';
-const replicaCount: string = '2';
+const storagePoolDropdown: string = 'pool-dropdown-toggle';
+
+const prepareStorageClassForm = () => {
+  // select provisioner
+  cy.byTestID('storage-class-provisioner-dropdown').click();
+  cy.byLegacyTestID('dropdown-text-filter').type('openshift-storage.rbd.csi.ceph.com');
+  cy.byTestID('dropdown-menu-item-link').should('contain', 'openshift-storage.rbd.csi.ceph.com');
+  cy.byTestID('dropdown-menu-item-link').click();
+
+  // select block pool
+  cy.byTestID(storagePoolDropdown).click();
+  cy.byTestID('create-new-pool-button').click();
+};
+
+const createBlockPool = (poolCreationAction: string) => {
+  // Make sure the storage pool creation form is open
+  modal.modalTitleShouldContain('Create New Block Pool');
```

Comment:
ack

Reply:
BlockPool is not a resource, it is a display name. CephBlockPool is the resource name.

In that case, i have change this to `Create block pool` 

including from OCS operator to display header

Reply:
I checked with core team, now everything changed to no space. So instead of `block pool` or `Block Pool`.
We have ti say `BlockPool`.
Please change this in `label` and `labelPlural` of `CephBlockPool` model as well.

Reply:
Let's create a different PR for fixing this.